### DISCUSSION TITLE
Feature Request: Allow dynamic workflow run titles

### DISCUSSION THREAD

**Discussion by jasonf20** [community/community#42703] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

We have lots of workflows that we run multiple times with different inputs. 

When trying to find a specific run and/or identify what a specific run was we have trouble locating it since all the build runs have the same name in the list. We would like to be able to push some of the info from the user input into the run name. 

Currently, we log the inputs into a summary file, but that means we still need to open each run. 

P.S. Being able to see the inputs the build was run with without having to manually log them would also be nice.

**Comment by dc185334** [comment, marked as answer]
I'm gonna try the run-name approach since we also need it:
https://github.blog/changelog/2022-09-26-github-actions-dynamic-names-for-workflow-runs/

**Reply by jasonf20** [reply]
Oh I didn't notice that feature being published, looks great will try it out. Thanks for the info!

**Reply by jasonf20** [reply]
Seems to work well 👍

**Reply by dc185334** [reply]
Yup, the same for me. 
Now my Fork RC contains RC number in a title, 
and it is not necessary to jump into the workflow log to catch it.
